### PR TITLE
shows interlock warnings, add buttons to expand warnings

### PR DESCRIPTION
**Pull request by pkubanek**

### CODE REVIEW

File: python/lsst/ts/cRIOpy/GUI/CustomLabels.py [hunk 1]
```diff
@@ -307,6 +333,96 @@ def setValue(self, value):
...
+    def _data(self, data):
+        self.setValue(getattr(data, self._field))
+
+    def setValue(self, value):
+        """Sets formatted value. Color codes WARNING (red)/OK (green).
+
+        Parameters
+        ----------
+        value : `bool`
+            Current (=to be displayed) variable value. False is OK, True is a problem.
```
Comment (teweitsai) [inline python/lsst/ts/cRIOpy/GUI/CustomLabels.py:418]: I think you could have a better name than `value`, which is a bool here. I thought the use of `value` word will be more wide than bool only.

File: python/lsst/ts/cRIOpy/GUI/EventWindow.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+from PySide2.QtCore import Slot
+from PySide2.QtWidgets import QWidget, QVBoxLayout, QTreeView
+from PySide2.QtGui import QStandardItemModel, QStandardItem
+
+
+class EventWindow(QWidget):
```
Comment (teweitsai) [inline python/lsst/ts/cRIOpy/GUI/EventWindow.py:27]: Please consider to improve the document of this class.

File: python/lsst/ts/cRIOpy/GUI/EventWindow.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        getattr(comm, topic).connect(self.newData)
+
+        data = getattr(comm.remote, "evt_" + topic).get()
+        if data:
+            self.newData(data)
+
+    @Slot(map)
+    def newData(self, data):
+        if self._data is None:
+            for a in data.__dict__:
```
Comment (teweitsai) [inline python/lsst/ts/cRIOpy/GUI/EventWindow.py:52]: Please consider to has a more meaningful name than `a`.

File: python/lsst/ts/cRIOpy/GUI/InclinometerPageWidget.py [hunk 4]
```diff
@@ -7,88 +7,44 @@
...
+        layout.addLayout(dataLayout)
+        layout.addSpacing(20)
+        layout.addWidget(
+            WarningGrid(
+                {
+                    "anyWarning": "Any Warnings",
+                    "sensorReportsIllegalFunctionLabel": "Sensor Illegal Function",
+                    "sensorReportsIllegalAddressLabel": "Sensor Illegal Address",
+                    "responseTimeout": "Reponse Timeout",
+                    "invalidCRC": "Invalid CRC",
```
Comment (teweitsai) [inline python/lsst/ts/cRIOpy/GUI/InclinometerPageWidget.py:32]: Please consider to add the comment to explain what is full name of `CRC`. No idea of this.

**Review by teweitsai — APPROVED** [review]
The PR looks good to me. Please consider to have some unit tests if possible. Thanks!

File: python/lsst/ts/cRIOpy/GUI/CustomLabels.py (hunk 1 above)
Comment (pkubanek) [reply python/lsst/ts/cRIOpy/GUI/CustomLabels.py:418]: Renamed to interlockOff

File: python/lsst/ts/cRIOpy/GUI/EventWindow.py (hunk 2 above)
Comment (pkubanek) [reply python/lsst/ts/cRIOpy/GUI/EventWindow.py:27]: Documented.

File: python/lsst/ts/cRIOpy/GUI/InclinometerPageWidget.py (hunk 4 above)
Comment (pkubanek) [reply python/lsst/ts/cRIOpy/GUI/InclinometerPageWidget.py:32]: CRC is Cyclic Redundancy Check. 
image

File: python/lsst/ts/cRIOpy/GUI/EventWindow.py (hunk 3 above)
Comment (pkubanek) [reply python/lsst/ts/cRIOpy/GUI/EventWindow.py:52]: image

**Pull request merged**